File: hard/9_traverse_graph/traverse_graph_test.py
import unittest
from traverse_graph import numberOfWaysToTraverseGraph

class TestTraverseGraph(unittest.TestCase):
    def test_case_1(self):
        width = 4
        height = 3
        expected = 10
        actual = numberOfWaysToTraverseGraph(width, height)
        self.assertEqual(actual, expected)

File: hard/9_traverse_graph/traverse_graph.test.js
const  numberOfWaysToTraverseGraph = require('./traverse_graph');

test("Test Case #1", () => {
  const width = 4;
  const height = 3;
  const expected = 10;
  const actual = numberOfWaysToTraverseGraph(width, height);
  expect(actual).toEqual(expected);
});
